ci-operator/step-registry/baremetalds/e2e/virtualmedia/baremetalds-e2e-virtualmedia-workflow.yaml

```diff
@@ -0,0 +1,19 @@
+workflow:
+  as: baremetalds-e2e-virtualmedia
+  steps:    
+    env:
+      DEVSCRIPTS_CONFIG: |
+        PROVISIONING_NETWORK_PROFILE=Disabled
```

Comment:
What do you think about making this also add `IP_STACK=v4`? Disconnected adds quite some time to a deployment. Is there a reason to do this as IPv6?

Reply:
I just copied the old conf file, but I agree it makes more sense

Reply:
Yea I didn't realize it was always IPv6. Thanks!